pkg/assembler/nodes.go

```diff
@@ -431,3 +459,58 @@ func (e MetadataForEdge) PropertyNames() []string {
 func (e MetadataForEdge) IdentifiablePropertyNames() []string {
 	return []string{}
 }
+
+// AttestationForPackage is an edge that represents the fact that an
+// `AttestationNode` is an attestation for an `PackageNode`.
+type AttestationForPackage struct {
```

Comment:
hmm wondering if we should do this as extending `AttestationFor` to be able to take either package or artifact... kind of like what we did for https://github.com/guacsec/guac/blob/main/pkg/assembler/nodes.go#L404-L413

Reply:
Yea I like that idea. Let me update.

Reply:
done.